aiosmtpd/tests/conftest.py

```diff
@@ -26,14 +26,22 @@
     _ProactorBasePipeTransport = None
     HAS_PROACTOR = False
 
+try:
+    import trustme
+
+    # Check if the CA is available in runtime, MacOS on Py3.10 fails somehow
```

Comment:
I don't know if this is still true. Let's drop the check and just keep the plain import.